Estrcutura de aplicacion movil

andersonsj opened the pull request:
Se genera estructura de aplicacion movil con carpetas para Backend y Fronend

Commit:
Se agrega estructura de carpetas

Commit:
Se agrega estructura backend aplicacion movil

Commit:
Se elimina READMIN sin uso

Commit:
Se agrega README con integranes edel grupo

RASP0905 committed:
Se agrega estructura front

RASP0905 committed:
Merge branch 'master' of https://github.com/cjmedina1/segubici

RASP0905 committed:
pantalla del login

Commit:
Se agrega modelos de backend

Commit:
Se agrega conexion MYSQL

cjmedina1 commented:
Se agrega el backend inicial de la aplicacion

RASP0905 committed:
Home page completo

RASP0905 committed:
Merge branch 'master' of https://github.com/cjmedina1/segubici

Commit:
Se agrega configuracion para modificar tablas desde administrador

Commit:
Merge branch 'master' of https://github.com/cjmedina1/segubici

fedesoft207 commented on Aplicacion/BackSeguBici/BackSeguBici/movil/models.py:
```diff
@@ -1,9 +1,40 @@
+    numerocalle = models.CharField(max_length=3)
+    letratres = models.CharField(max_length=1)
+    numerocomplemento = models.CharField(max_length=3)
+    cuidad = models.ForeignKey(Ciudad, on_delete=models.CASCADE)
```
Arreglar nombre de campo

fedesoft207 commented on Aplicacion/BackSeguBici/BackSeguBici/movil/models.py:
```diff
@@ -1,9 +1,40 @@
+    descripcioncardinalidad = models.CharField(max_length=30)
+
+class Departamento(models.Model):
+    descrpciondepartamento = models.CharField(max_length=30)
```
Arreglar nombre de campo y dejar cualquier campo de descripcion de tipo models.TextField()

fedesoft207 commented on Aplicacion/BackSeguBici/BackSeguBici/movil/models.py:
```diff
@@ -1,9 +1,40 @@
-
-class imagen (models.Model):
+class Imagen (models.Model):
     urlimagen = models.CharField(max_length=50)
```
Revisar el tutorial Campos Django para el campo de imagen

RASP0905 committed:
creacion de la pagina de rergistrarse y de sus respectivos campos y se comienza a realizar codigo para cargar foto

RASP0905 committed:
Merge branch 'master' of https://github.com/cjmedina1/segubici

cjmedina1 committed:
Ajuste de imagen de fondo

Commit:
Se agraga servicion para cargar imagen bicicleta

Commit:
Se agrega servicio de login usuarios

RASP0905 committed:
se agrega tomar foto y se prueba en un celular

RASP0905 committed:
Merge branch 'master' of https://github.com/cjmedina1/segubici

Commit:
Se agrega servicio de registro

Commit:
Se agrega servicio de registro

Commit:
Creacion de gitignore

RASP0905 committed:
cambio de campos al registrarse

cjmedina1 committed:
se agrega Login y se ajusta fondo de pantalla de inicio

cjmedina1 committed:
Login

cjmedina1 committed:
Login

cjmedina1 committed:
ajuste arch. rest

RASP0905 committed:
se genera la pagina de registro con la interaccion hacia la base de datos, tambien se agregan serilaizer a backend y la lista desplegable de la de la pagina de registro en el campo de tipo de identificacion

RASP0905 committed:
se genera la pagina de registro con la interaccion hacia la base de datos, tambien se agregan serilaizer a backend y la lista desplegable de la de la pagina de registro en el campo de tipo de identificacion

cjmedina1 committed:
Se crea la pagina de Ubicación (Geolocalización) y se configura como página de aterrizaje para el login.

RASP0905 committed:
se cambia la ubicacion del tipo de identificacion

RASP0905 committed:
Merge branch 'master' of https://github.com/cjmedina1/segubici

Commit:
Se genera relacion entre propietario y localizacion

RASP0905 committed:
Merge branch 'master' of https://github.com/cjmedina1/segubici

RASP0905 committed:
se genera el registrarse, el cerrar sesion y se crea la funcion loading

Commit:
Se agregan json para poblar tablas ejecutar comandos en archivo Comandospoblarbase

Commit:
Se agregan json para poblar tablas ejecutar comandos en archivo Comandospoblarbase

Commit:
Merge branch 'master' of https://github.com/cjmedina0/segubici

RASP0905 committed:
se genera el mapa de ubicacion, aunque falta probarlo en un celular

cjmedina1 committed:
se crea pagina ubicacion

cjmedina1 committed:
Merge branch 'master' of https://github.com/cjmedina1/segubici

Commit:
Se agregan servicios Bicicleta,Tipocalle,Cardinalidad,Departamento,Ciudad,Localizacion,Geolocalizacion

Commit:
Se agrega url de servicios

Commit:
Se agrega pagina de registro de direccion

RASP0905 committed:
se esta generando la pagina para registrar la pagina

RASP0905 committed:
se esta generando la pagina para registrar la bici

cjmedina1 committed:
Se elimina el botón de ubicacion. Muestra el mapa y la ubicacion directamente.
Se agregan puntos de zonas de peligro
Se agrega mensaje de alerta "Estas en una zona peligrosa"

cjmedina1 committed:
Ubicacion con puntos

RASP0905 committed:
se genera la pagina de registrar bicicleta con su respectiva lista desplegable

RASP0905 committed:
se genera la pagina de registrar bicicleta con su respectiva lista desplegable

RASP0905 committed:
se genera el boton para conformar el registro exitoso de la bici y cuando se presiona ok se devuelve a la pantalla de ubicacion

RASP0905 committed:
error

RASP0905 committed:
error git

RASP0905 committed:
se genera la subida de la foto, y alerta tost para cuando se ingresa un usuario o contraseña errados

cjmedina1 committed:
correccion de ubicacion
